src/test/java/io/xstefank/wildfly/bot/utils/JsonHandler.java

```diff
@@ -0,0 +1,141 @@
+package io.xstefank.wildfly.bot.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Class responsible for handling and changing properties of the JSON template file.
+ */
+public class JsonHandler {
```

Comment:
Also, since this is GH specific, wouldn't the name like GitHubJson make more sense?